blog/source/_posts/MIDI-velocity-transformer.md

```diff
@@ -0,0 +1,266 @@
+---
+title: MIDI Velocity Prediction with Transformer
+date: 2023-09-11
+tags:
+author: wmatejuk
+---
+
+### Why is it important?
+
+{% algrtmImgBanner MIDI-velocity-transformer/banner-1.jpg pianoroll %}
+
+MIDI velocity is a crucial element in music dynamics, determining the force with which a note is played, 
+which profoundly influences the emotional quality of music. 
+
+If you were to take a sequence of notes and predict their velocities by an untrained model, this is what you wold have ended up with:
+<div style="-webkit-column-count: 2; -moz-column-count: 2; column-count: 2;">
+    
+  **Original** 
+        
+  **Predicted by untrained model**
+
+</div>
+
+
+<div style="-webkit-column-count: 2; -moz-column-count: 2; column-count: 2;">
+
+  {% algrtmImg MIDI-velocity-transformer/samples/9115-real.png pianoroll 170px %}
+  {% algrtmAudio MIDI-velocity-transformer/samples/9115-real.mp3 %}
+        
+  {% algrtmImg MIDI-velocity-transformer/samples/9115-pred-untrained.png pianoroll 170px %}
+  {% algrtmAudio MIDI-velocity-transformer/samples/9115-pred-untrained.mp3 %}
+
+</div>
+
+
+Sounds pretty bad, doesn't it?
+
+Our Transformer-based model aims to decode 
+this nuanced aspect of musical expression, unraveling the hidden patterns 
+within quantized MIDI data.
+
+
+### Model Overview
```

Comment:
I think we don't need a model overview at all here. We can assume that the readers know what a transformer is, and we can include some references in the text.

What this and the next section is missing for me, is a description of how we convert the piano performance problem into a list of tokens problem.